tests/python/direct/test_repro.py

```diff
@@ -6,6 +6,7 @@
 import torch
 import pytest
 from nvfuser_direct import FusionDefinition, DataType
+from conftest import require_device_mem_size_gb
```

Comment:
conftest is for pytest fixtures, not regular testing utilities. 

Consider `from python.direct_utils`